sdb/commands/address.py

```diff
@@ -42,9 +42,11 @@ class Address(sdb.Command):
 
     names = ["address", "addr"]
 
-    def _init_argparse(self, parser: argparse.ArgumentParser) -> None:
-        super()._init_argparse(parser)
+    @classmethod
+    def _init_parser(cls, name: str) -> argparse.ArgumentParser:
+        parser = super(Address, cls)._init_parser(name)
```

Comment:
It's a bit annoying that we have to do it like that, is there a reason why the following doesn't work:
```
parser = super()._init_parser(name)
```
Also if you actually gave the above a try is there any advantage on using the more verbose version that you have?

Reply:
I can try that; I can't recall if I had tried that or not. This is a class method, though, so I think we need to do it this way. But I'll double check.. I also would much rather just use `super()` if possible.

Reply:
It looks like `super()` does work, so I don't know why I didn't try that to begin with. I guess my lack of experience with Python is showing :)